xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm

```diff
@@ -139,7 +139,7 @@
   #end
   #template("header.vm")
   #if($editor == 'wiki' || $editor == 'wysiwyg')
-    <form id="edit" method="post" action="$doc.getURL('preview')" class="withLock form">
+    <form id="edit" method="post" action="$doc.getURL('preview')" class="withLock xform">
```

Comment:
You didn't just added the ``xform`` class, you also **removed** the ``form`` class. We may loose styles as a consequence. Did you check if there were styles associated with the ``form`` class? You can also break any JavaScript code that looks for the ``form`` class. Did you check if there is any such JavaScript code?

Reply:
I just checked in lock.js :
// The page is automatically unlocked when the form is submitted.
    var markUnlocked = this.setLocked.bind(this, false);
    $$('form.withLock').each(function(form) {
      form.observe('submit', markUnlocked);
    });
I am not sure if this requires 'form' coming after withLock.
So just in case I added xform after 'form'. It also worked as intended.
Thanks for letting me know my mistake.

Reply:
``form.withLock`` is indeed a CSS selector, but ``form`` is not a class name but the tag name. So this JavaScript code is not a good reason to keep the ``form`` class name.

Reply:
So to be clear: I'm not saying you should keep the ``form`` class name. I'm just saying that if you remove it you should check if there are CSS styles that we may loose or if there is some JavaScript code that you may break.

Reply:
Ok got it! A little question: If it breaks, I will get an error or the styling would be messed up right?